src/index.html

```diff
@@ -1,16 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="#"
+      data-qa="big"
+    >
+      <div class="input">
+        <img
+          src="images/Search.svg"
+          alt="search-icon"
+          class="icon"
+        >
+        <input
+          type="text"
+          class="inputfield"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
```

Comment:
```suggestion
        <input
          type="text"
          class="search-form__input"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```

Reply:
in BEM you should use classnaming with structure
{block}__{element}_{modificator}
https://en.bem.info/methodology/quick-start/